spring-boot-project/spring-boot-autoconfigure/src/main/java/org/springframework/boot/autoconfigure/kafka/KafkaProperties.java

```diff
@@ -1314,6 +1321,119 @@ public void setOptions(Map<String, String> options) {
 
 	}
 
+	public static class Retry {
+
+		private Topic topic = new Topic();
+
+		public Topic getTopic() {
+			return this.topic;
+		}
+
+		public void setTopic(Topic topic) {
+			this.topic = topic;
+		}
+
+		/**
+		 * Properties for non-blocking, topic-based retries.
+		 */
+		public static class Topic {
+
+			/**
+			 * Whether to enable topic-based retries auto-configuration.
+			 */
+			private Boolean enabled;
+
+			/**
+			 * The total number of processing attempts made before sending the message to
+			 * the DLT.
+			 */
+			private Integer attempts = 3;
+
+			/**
+			 * A canonical backoff period. Used as an initial value in the exponential
+			 * case, and as a minimum value in the uniform case.
+			 */
+			private Duration delay = Duration.ofSeconds(1);
+
+			/**
+			 * If positive, then used as a multiplier for generating the next delay for
+			 * backoff.
+			 */
+			private Double multiplier = 0.0;
+
+			/**
+			 * The maximimum wait between retries. If less than the delay then the default
+			 * of 30 seconds is applied.
+			 */
+			private Duration maxDelay = Duration.ZERO;
+
+			/**
+			 * In the exponential case (multiplier() > 0) set this to true to have the
+			 * backoff delays randomized, so that the maximum delay is multiplier times
+			 * the previous delay and the distribution is uniform between the two values.
```

Comment:
A description for these configuration properties must be as short as possible: https://docs.spring.io/spring-boot/docs/current/reference/htmlsingle/#appendix.configuration-metadata.format.property

Reply:
Sure. This description was extracted from the `@BackOff` annotation, I'll try to come up with something more succinct. Thanks.